packages/library/src/commonMain/kotlin/io/realm/Queryable.kt

```diff
@@ -21,6 +21,7 @@ package io.realm
 //    dependent on how the final API is going to look like.
 //  - Query could alternatively be separated into builder to await constructing new results until
 //    actually executing the query
+// FIXME Is this deprecated?
```

Comment:
It is part of #206 to revisit this

Reply:
Ok. Updated with a comment that this might change and a reference to the issue.